modin/pandas/groupby.py

```diff
@@ -135,7 +136,20 @@ def __init__(
         }
         self._kwargs.update(kwargs)
 
-    def __override(self, **kwargs):
+    def _override(self, **kwargs):
```

Comment:
Made this method accessible out of this class so we can call it from `RollingGroupby`